Library/Formula/conman.rb

```diff
@@ -0,0 +1,24 @@
+class Conman < Formula
+  desc "ConMan: The Console Manager"
+  homepage "https://github.com/dun/conman"
+  url "https://github.com/dun/conman/archive/conman-0.2.7.tar.gz"
+  sha256 "bf96dfbb05e1592355bab3afda4fcb4766a57bca8bc98eb1dd5020f24039d39f"
+
+  depends_on :x11 => :optional
```

Comment:
There's probably an explicit `--without-x11` flag we can add when `build.without? "x11"` is true.